packages/block-library/src/tag-cloud/block.json

```diff
@@ -37,6 +37,12 @@
 	"supports": {
 		"html": false,
 		"align": true,
+		"color": {
+			"text": true,
```

Comment:
Is there much value in enabling both text and link color support for Tag Cloud? 

I suppose it could be possible for a theme or plugin to filter `wp_generate_tag_cloud` to alter the markup in a way that would include non-link text. That said, I'm not sure it is something we need to cater for though.

Reply:
Now that I see it, we can probably rule out `text` support because there will just be links. Ill update the PR.

Reply:
Apologies @akasunil, I might have been wrong to question the text color support. It's just clicked for me that the Tag Cloud block does render simple text when there are no tags to display. 

It's conceivable that a user might wish to ensure the text color works nicely with the background color support added here. What do you think?

Reply:
@aaronrobertshaw I've opened a [PR](https://github.com/WordPress/gutenberg/pull/63774) to prevent empty tag cloud blocks from rendering on the frontend. So that placeholder text will be visible in editor only when tags are not available.